plugins/chat/spec/support/api/schemas/user_chat_channel_membership.json

```diff
@@ -24,7 +24,6 @@
         "name": { "type": "string" },
         "avatar_template": { "type": "string" },
         "username": { "type": "string" },
-        "custom_fields": { "type": ["object", "null"] },
```

Comment:
This was previously required when I relied on `ChatableUserSerializer` but it's no longer needed.